pkg/output/duration.go

```diff
@@ -0,0 +1,18 @@
+package output
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/duration"
+)
+
+// TranslateTimestampSince returns the elapsed time since timestamp in
+// human-readable approximation.
```

Comment:
This is based on translateTimestampSince. So the duration matches the one kubectl shows.

https://github.com/kubernetes/cli-runtime/blob/v0.21.0/pkg/printers/tableprinter.go#L564-L572